xAPI-Communication.md

```diff
@@ -475,6 +467,9 @@ Stores a single Statement with the given id. POST can also be used to store sing
 
 ###### LRS Requirements
 
+* While an LRS MUST NOT have duplicate Statement Ids due to the uniqueness requirement, it does not have to perform 
+checks at the time Statements are received.  The following requirements are for LRSs that do such a check:
```

Comment:
I was trying to add clarity, but I see that I may have opened up a hole in the existing logic.  I removed.